src/graphql.rs

```diff
@@ -3541,6 +3555,8 @@ impl ___Type for FilterTypeType {
                             default_value: None,
                             sql_type: None,
                         },
+                        // shouldn't happen since we've covered all cases in supported_ops
+                        _ => panic!("encountered unknown FilterOp")
```

Comment:
The Scalar's `supported_ops` won't contain any of the array-only filter operations, so we're safe to ignore them here.

Reply:
we lean on the compiler to help remember where to update everything and a default case will prevent that from working. Please enumerate them separately even if they all throw the same error

Reply:
Maybe we should restructure the `FilterOp` and related types so that we can avoid this catchall panic. As this PR is already long enough, I'm ok with doing that as a separate PR. Just open a ticket to handle this later.